src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Infrastructure/OuterApi/Requests/Cohorts/GetCohortDetailsRequest.cs

```diff
@@ -18,10 +23,34 @@ public GetCohortDetailsRequest(long providerId, long cohortId)
 
     public class GetCohortDetailsResponse
     {
+        public long CohortId { get; set; }
+        public string CohortReference { get; set; }
+        public long AccountId { get; set; }
+        public long AccountLegalEntityId { get; set; }
         public string LegalEntityName { get; set; }
         public string ProviderName { get; set; }
+        public long? ProviderId { get; set; }
+        public bool IsFundedByTransfer => TransferSenderId.HasValue;
+        public long? TransferSenderId { get; set; }
+        public int? PledgeApplicationId { get; set; }
+        public Party WithParty { get; set; }
+        public string LatestMessageCreatedByEmployer { get; set; }
+        public string LatestMessageCreatedByProvider { get; set; }
+        public bool IsApprovedByEmployer { get; set; }
+        public bool IsApprovedByProvider { get; set; }
+        public bool IsCompleteForEmployer { get; set; }
+        public bool IsCompleteForProvider { get; set; }
+        public ApprenticeshipEmployerType LevyStatus { get; set; }
+        public long? ChangeOfPartyRequestId { get; set; }
+        public bool IsLinkedToChangeOfPartyRequest => ChangeOfPartyRequestId.HasValue;
+        public TransferApprovalStatus? TransferApprovalStatus { get; set; }
+        public LastAction LastAction { get; set; }
+        public bool ApprenticeEmailIsRequired { get; set; }
         public bool HasNoDeclaredStandards { get; set; }
         public bool HasUnavailableFlexiJobAgencyDeliveryModel { get; set; }
         public IEnumerable<string> InvalidProviderCourseCodes { get; set; }
+        public IReadOnlyCollection<DraftApprenticeshipDto> DraftApprenticeships { get; set; }
```

Comment:
We should be moving away from types as defined in this package, and not adding more references to them. The point of the outer api is to decouple the UI from the inner api, and by extension any packages that define types exposed by that api. It seems odd to invoke one api, but depend on the definition of types of another api.

Reply:
Thanks @chrisfoster76, how do you suggest I go about fixing/removing this type?

Reply:
I'm not suggesting you remove it from everywhere, just not to add it here.

Reply:
We have removed it from here and the ApprenticeEmailOverlap type and the DeliveryModel type